app/models/record.server.ts

```diff
@@ -9,34 +9,44 @@ export async function getRecordsByUsername(username: Record['username']) {
 }
 
 export async function createRecord(
-  username: Record['username'],
-  name: Record['name'],
-  type: Record['type'],
-  value: Record['value'],
-  status: Record['status'],
-  description?: Record['description'],
-  course?: Record['course'],
-  ports?: Record['ports']
+  data: Pick<Record, 'username' | 'name' | 'type' | 'value' | 'status'>
 ) {
   // Set expiration date 6 months from now
   const expiresAt = new Date();
   expiresAt.setMonth(expiresAt.getMonth() + 6);
 
-  return prisma.record.create({
-    data: {
-      username,
-      name,
-      type,
-      value,
-      description,
-      course,
-      ports,
-      expiresAt,
-      status,
-    },
-  });
+  return prisma.record.create({ data: { ...data, expiresAt } });
 }
 
+// export async function createRecord(
```

Comment:
Please remove dead code. Git never forgets, so there is no need to keep it here
I suggest using the "Git Lens" plugin in VsCode if you have not encountered it yet

Reply:
Oh, sorry about that. Seems like I missed this